src/coap_openssl.c

```diff
@@ -1964,6 +1981,8 @@ int coap_dtls_receive(coap_session_t *session,
     int err = SSL_get_error(ssl, r);
     if (err == SSL_ERROR_WANT_READ || err == SSL_ERROR_WANT_WRITE) {
       if (in_init && SSL_is_init_finished(ssl)) {
+        coap_log(LOG_CIPHERS, "*  %s: Using Cipher: %s\n",
```

Comment:
I think the word "cipher" in this and the following log messages should be lower case.

Reply:
Updated all Ciphers to ciphers.